.github/workflows/build-rad-ui.yml

```
@@ -7,21 +7,77 @@
       - 'styles/**'
       - 'package.json'
+
+permissions:
+  contents: read
+
 jobs:
-  build:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      node-modules-cache: ${{ steps.cache.outputs.cache-hit }}
```

actionlint (1.7.7)

property "cache" is not defined in object type {}

(expression)